apps/condo/bin/create-b2bapp.js

```diff
@@ -15,7 +16,7 @@ function getJson (data) {
 }
 
 async function main (args) {
-    const [name, options] = args
+    const [name, options, appName] = args
```

Comment:
What is the difference between B2BApp `name` and `appName`?

Reply:
Looks like the same thing

Reply:
`appName` it's a name of miniapp (like `insurance`, `pass` etc), name – it's a name of related B2BApp. They are can be different if we need

Reply:
ok, then it is better to use `b2bAppName` rather than `name`